lib/cash-in/cash-in-atomic.js

```diff
@@ -46,8 +46,15 @@ function insertNewBills (t, billRows, machineTx) {
   const dbBills = _.map(cashInLow.massage, bills)
   const columns = _.keys(dbBills[0])
   const sql = pgp.helpers.insert(dbBills, columns, 'bills')
-
-  return t.none(sql)
+  const deviceID = machineTx.deviceId
+  const sql2 = `update devices set cashbox = cashbox + 1
```

Comment:
Are we sure this is only one at a time? Can you try shutting down the server after the first bill insertion -> insert a couple more bills -> turn the server back on.

Reply:
I was thinking, wouldn't it be cleaner to add dbBills.length? Would solve that problem and would be more understandable, as it is not a number spawned out of nowhere